main.js

```diff
@@ -0,0 +1,213 @@
+// 1.
+/*Створити функцію-конструктор Людини.
+ Властивості, які будуть описувати екземпляр Людини - ім'я, вік, стать, національність, країна, список країн для подорожей.
+ Створити універсальні функції - знайомство, прокидатися, засинати й список країн,
+ які Людина хоче відвідати.Роботу цих функцій можна виводити через console.log.Але
+ в кожній з функцій повинно використовуватись якомога більше
+ характеристик конкретної Людини на якій ця функція викликається(підказка: тут треба використатии call / apply / bind) */
+
+function CreateHumanList(name, age, sex, nationality, country, []) {
+  this.nameOfHuman = name;
+  this.ageOfHuman = age;
+  this.sexOfHuman = sex;
+  this.nationalityOfHuman = nationality;
+  this.countryOfHuman = country;
+  this.countryToTravel = [];
+}
+
+const humanMe = new CreateHumanList(
+  "Vladyslav",
+  34,
+  "Male",
+  "Ukrainian",
+  "Ukraine",
+  ["Italy", "Spain", "Poland", "Portugal"]
+);
+
+const greet = function () {
+  console.log(`Hello ${this.nameOfHuman} have a nice day!`);
+};
+greet.apply(humanMe);
+
+const wakeUp = function () {
+  console.log(`Wake up ${this.nameOfHuman} already, you're late for work!`);
+};
+wakeUp.apply(humanMe);
+
+const sleep = function () {
+  console.log(
+    `I'm very tired today, I'm going to sleep and dream about ${["Portugal"]}.`
+  );
+};
+sleep.apply(humanMe);
+
+const travel = function () {
+  console.log(
+    `At my ${this.ageOfHuman} years old, I plan to go ${[
+      "Poland",
+    ]} and they will welcome me there because I am ${this.nationalityOfHuman}.`
+  );
+};
+travel.apply(humanMe);
+
+// // 2.
+// /*  Створити власну реалізацію методу.bind */
+
+function bind(fn, context, ...rest) {
+  return function (...args) {
+    const uuid = Date.now().toString();
+    context[uuid] = fn;
+    const res = context[uuid](...rest, ...args);
+    delete context[uuid];
+    return res;
+  };
+}
+const cardOfDog = {
+  breedOfDog: "",
+  nameOfDog: "",
+  fullInfo() {
+    return `I love my ${this.breedOfDog} his named is ${this.nameOfDog}.`;
+  },
+};
+
+function getFullProfile(breedOfDog, nameOfDog) {
+  this.breedOfDog = breedOfDog;
+  this.nameOfDog = nameOfDog;
+  return this.fullInfo();
+}
+
+bind(getFullProfile, cardOfDog, "Amstaff", "Oscar")();
```

Comment:
According to `.bind` from JS we have accepts the 1-st argument as the new context, which should always be evaluated to object and an optional list of arguments, separated by a comma

In your case - bind accepts a function, object, and 2 strings, and the main idea is not realized - we do not bound context using apply or call